### DISCUSSION TITLE
gRPC

### DISCUSSION THREAD

**Discussion by ibrahim-softsuave** [getezy/ezy#51] [Q&A] [answered]
I have a grpc client in my local node server and I have grpc server in google cloud 
how can i generate a certificates for these constraints.
since i have idea about generate local certificate but for server i dont have any idea

**Comment by notmedia** [comment, marked as answer]
@ibrahim-softsuave 
Hello! You can read this [article](https://itnext.io/how-to-setup-and-test-tls-in-grpc-grpc-web-1b67cc4413e6), I think this should help to understand how to configure TLS in your app.

Anyway if you want to work with TLS encryption you should install certificate on you google cloud server. It could be done with setting up cert inside your server's source code where you creating gRPC server.
